bench/triton_bench/matmul_ogs_details/_ptma_matmul_ogs.py

```diff
@@ -368,7 +367,11 @@ def _ptma_matmul_ogs(
                     mask_k = tl.arange(0, BLOCK_K) < K - off_k
                     x = tl.load(XPtrs, mask=mask_k[None, :], other=0.0)
 
-            w = _load_tensor_desc(w_desc, [expt_id, off_k_w, off_n], transpose=W_TRANSPOSE)
+            # Note: use off_k here which spans BLOCK_K elements and _not_ PACKED_BLOCK_K_W bytes.
+            # When using the unpacked destination dtype, the fp4 elements are unpacked into a padded
+            # layout, b4x16_p64. In this case, the TMA requires the contiguous dimension be a multiple
+            # of 128 elements.
```

Comment:
This is wrong, the descriptor API has int8 dtypes so we should be offsetting in bytes here and the TMA lowering should be doing the conversion to fp4 elements for us.

Reply:
If this is only needed when enabling warp specialization then there must be a bug elsewhere.

Reply:
Looks like this function isn't calling `translateTMAIndices`
https://github.com/triton-lang/triton/blob/3bd4a58ae718f7383737b05bd8a6531f7a76d9c6/lib/Dialect/TritonGPU/Transforms/WarpSpecialization/LoadMMASpecialization.cpp#L142

Reply:
Ooh, thank you for the great observation! I've removed this and updated LoadMMASpecialization to use the translated indices.